### ISSUE TITLE
Support for multiple datasets

### ISSUE THREAD

**Issue by astrofrog** [glue-viz/glue-pyspeckit-plugin#1]
Datasets could be selected in a combo above the component combo

**Comment by keflavich** [comment]
Proposed UI: Dropdown menu in the right-side panel that selects the dataset out of the listed datasets, and that one will be fit.

Both (all) spectra will have to be plotted together.  The spectrum selected to be fitted should be highlighted, perhaps in blue?

**Comment by astrofrog** [comment]
Yes, that could work - and in fact we can also show each spectrum as a plot layer so that their visual properties can be edited there too.
